themes/skyline/html/email.html.twig

```diff
@@ -57,7 +57,7 @@
 
     <style type="text/css" media="screen">
         @media screen {
-             /* Thanks Outlook 2013! http://goo.gl/XLxpyl*/
+             /* Thanks Outlook 2013! dropped shortlink t-online spam detection for shortlinks*/
```

Comment:
Could we just remove the text all together after the exclamation mark? I don't think it's necessary to retain anything there.

Reply:
i think yes. Maybe the developer wont get lost the documentation. So i am new here. MayBe a readme for themes? i dont know if it possible.